.github/workflows/static.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           composer-options: "--ignore-platform-reqs" # required for PHP 8.2 as not yet official supported
 
       - name: Psalm
-        run: psalm --no-progress --output-format=github
+        run: psalm --php-version=8.2 --no-progress --output-format=github
```

Comment:
I'm unfamiliar with pslam @Nyholm can you have a look at this change if its required?

Reply:
I think this is inferred from the PHP version we are running on. 

I dont mind adding it, but it would be nice to hear what value it brings.

Reply:
@msmakouz can you answer that?

Reply:
@alexander-schranz @Nyholm In the Psalm action, PHP 8.2 is used, and the latest dependencies are used. However, without this option, Psalm checks with the PHP version specified as the minimum in composer.json, and it displays two errors:

```
Error: src/roadrunner-nyholm/src/Runner.php:23:19: UndefinedClass: Class, interface or enum named Spiral\RoadRunner\Worker does not exist (see https://psalm.dev/019)
Error: src/roadrunner-symfony-nyholm/src/Runner.php:35:19: UndefinedClass: Class, interface or enum named Spiral\RoadRunner\Worker does not exist (see https://psalm.dev/019)

```

https://github.com/php-runtime/runtime/actions/runs/9156994654/job/25172484593

I assume this is because the package **spiral/roadrunner-worker** (v3.6.0) requires a minimum PHP version of **8.1**. If this option is added, these errors are not present. Need to specify this option or lower the PHP version in the action to 8.0 so that the dependencies requiring a minimum PHP version of 8.0 are installed.

Reply:
Sounds for me like we are missing the requirement to `spiral/roadrunner-worker` package in our own `composer.json` also.

Reply:
The issue with Psalm is not related to this. This package was installed anyway because it is required by `roadrunner-php/http`. However, `spiral/roadrunner-worker` is used in this package, so it needs to be added, you are right. I have added it.